src/activity/activity.jsx

```diff
@@ -82,14 +84,68 @@ class DailyActivity extends Component {
     ]
   }
 
+  stdDeviation (data, average, getData = d => d) {
+    const diffs = data.map(d => Math.pow(getData(d) - average, 2))
+    return Math.sqrt(_mean(diffs))
+  }
+
+  aggregateFeatures (data, key, count) {
+    const getCount = d => d[count]
+    const getKey = d => d[key]
+
+    const distAvgToStdDev = _reduce(
+      this.state.data,
+      (result, d, key) => {
+        const values = (getKey(d) && getKey(d)) || []
+
+        const average = _meanBy(getKey(d), getCount)
```

Comment:
How about using `mean` everywhere instead of `average`? (since they are synonyms)